system-tests/__snapshots__/component_testing_spec.ts.js

```diff
@@ -564,3 +564,144 @@ exports['React major versions with Webpack executes all of the tests for React v
 
 
 `
+
+exports['experimentalSingleTabRunMode / executes all specs in a single tab'] = `
+We detected that you have versions of dependencies that are not officially supported:
+
+ - \`webpack\`. Expected >=4.0.0 || >=5.0.0 but dependency was not found.
```

Comment:
Does this mean our system test isn't in a great state if we arn't testing against our supported versions?

Reply:
What's happening is we are falling back to the bundled version of webpack that we ship with Cypress. In open mode we actually show a warning in the launchpad. In run mode, we don't do that.

We could add `webpack` to the `package.json` for this test project, if we'd like to get rid of this warning.